### PR TITLE
[SDK-239] Fix dataset types

### PR DESCRIPTION
**Pull request by mnoszczak**
Makes the types more correct

### CODE REVIEW

**Review by apollonin — APPROVED** [review]

**Review by kopreschko — APPROVED** [review]
lgtm

**Review by vbrodsky — COMMENTED** [review]
Not sure we need Optional here... please see this too https://stackoverflow.com/questions/69440494/python-3-10-optionaltype-or-type-none#:~:text=Optional%20was%20invented%20to%20indicate,gets%20rid%20of%20an%20identifier.

**Review by vbrodsky — COMMENTED** [review]
Also here we have a default None, so Optional is not required:
https://docs.python.org/3/library/typing.html#typing.Optional

File: labelbox/schema/dataset.py [hunk 1]
```diff
@@ -68,8 +68,8 @@ class Dataset(DbObject, Updateable, Deletable):
 
     def data_rows(
         self,
-        from_cursor: str = None,
-        where: Comparison = None,
+        from_cursor: Optional[str] = None,
```
Comment (vbrodsky) [inline labelbox/schema/dataset.py:71]: this is tricky, but correct change I believe, based on https://docs.python.org/3/library/typing.html#typing.Optional

**Review by vbrodsky — APPROVED** [review]

**Pull request merged**